environment/git/install-hooks.sh

```diff
@@ -13,7 +13,7 @@
 ###########################################################
 # CONFIGURATION:
 # select which pre-commit hooks are going to be installed
-HOOKS="pre-commit pre-commit-clang-format pre-commit-flake8 pre-commit-fprettify"
+HOOKS="pre-commit pre-commit-clang-format pre-commit-autopep8 pre-commit-flake8 pre-commit-fprettify"
```

Comment:
Do we still need  `pre-commit-flake8` if we adopt `pre-commit-autopep8`?

Reply:
If we don't mind switching to `autopep8` from `flake8`, that would solve the above consistency issue I brought up. I do like that `autopep8` will automatically apply formatting changes, but I don't know if there are reasons for us to still prefer `flake8` at the CI level

Reply:
The [superlinter](https://github.com/github/super-linter) CI check that we use will run `flake8`.  I want to keep that CI because it has a very low maintenance cost.  It sounds like we need to keep both for now.